packages/quantic/force-app/main/default/lwc/quanticTabBar/quanticTabBar.js

```diff
@@ -390,4 +390,27 @@ export default class QuanticTabBar extends LightningElement {
     // @ts-ignore
     return Math.ceil(element.offsetWidth + padding);
   }
+
+  /**
+   * Returns the tab bar dropdown container element.
+   * @returns {Element}
+   */
+  get tabBarDropdownContainer() {
+    return this.template.querySelector('.tab-bar_dropdown');
+  }
+
+  /**
+   * @param {FocusEvent} event
+   */
+
+  onBlur(event) {
+    if (
+      this.isDropdownOpen &&
+      event.relatedTarget instanceof HTMLElement &&
+      this.tabBarDropdownContainer.contains(event.relatedTarget)
+    ) {
+      return;
+    }
+    this.closeDropdown();
```

Comment:
I think reverting the condition would make the code easier to read. Perhaps you could also extract the part checking whether the target element is in the dropdown.

```suggestion
    const isTargetElementOutsideDropdown = !(event.relatedTarget instanceof HTMLElement &&
      this.tabBarDropdownContainer.contains(event.relatedTarget));
      
    if (this.isDropdownOpen && isTargetElementOutsideDropdown) {
      this.closeDropdown();
    }
```